next-css.patch

```diff
@@ -0,0 +1,40 @@
+diff --git a/css-loader-config.js b/css-loader-config.js
+index 19815a132c7a3ca5d77cba5cf327fdca428de517..be57e4fa37617c6ff11ba65f840b0f60a07f578f 100644
+--- a/css-loader-config.js
++++ b/css-loader-config.js
+@@ -70,7 +70,7 @@ module.exports = (
+     )
+ 
+     postcssLoader = {
+-      loader: 'postcss-loader',
++      loader: require.resolve('postcss-loader'),
```

Comment:
The legacy next-css package doesn't use `require.resolve` to access its loaders, which leaves it vulnerable when the hoisting changes slightly. In this case, it caused crashes because the `css-loader` version hoisted to the top wasn't exactly the same one as the one in dependency.

Since next-css is kinda legacy, and since it's just a devDependency, I figured it would be simpler to just apply the patch on this repository and be done with it (rather than having to publish a new version, upgrade it, etc).

Reply:
Just saying, wouldn't it be better for organization to put this in a separate directory just for patches?  My personal philosophy (my opinion, might not be reflected by project maintainers) is that the root directory is very sacred and should only contain essential files.  While this is essential, most projects that use patches put them in the `patches` directory.

Just my two cents.